### DISCUSSION TITLE
Webpage zoom level always reset when cookie auto-delete plugin is enabled, even if site is whitelisted.

### DISCUSSION THREAD

**Discussion by stemid** [Cookie-AutoDelete/Cookie-AutoDelete#987] [Q&A / Support] [answered]
How can I prevent Cookie auto-delete from resetting my website zoom level?

It's annoying to have to zoom in every website over and over whenever I re-visit. 

I have the site whitelisted, all checkboxes checked and it still resets the zoom level at next visit.

**Comment by kennethtran93** [comment]
1. Which browser (+ version) are you using?
2. Is this a browser zoom function or site zoom function?
3. We're you enabling the global site data cleanup in CAD settings (not the popup) only once or every time?

**Reply by stemid** [reply]
Sorry it's Firefox 82.0.2 on Fedora 33.

I use ctrl++ and Ctrl+- so I assume it's a browser feature. It's always been like this.

I haven't changed any settings, just looked at them now and I can't identify any such data cleanup setting. I tried resetting all the CAD settings to defaults. Only thing I changed is the first box that says "Enable automatic deletion" (translated from Swedish locale).

**Comment by kennethtran93** [comment, marked as answer]
Is this still happening even when CAD is turned off / autoclean off? 

It's a browser feature, to which the browser should keep the zoom settings per page/domain.

Maybe easier is to install an extension - I think there is one to have zoom set per tab or domain. 

Other people have said that if you are using multiple windows it may not work as intended from just using browser functions. Same with accessing same site between containers. 

Also apparently if the resist fingerprinting option is enabled the browser zoom consistency may not work.

**Reply by stemid** [reply]
Thank you! True enough I had gone through the about:config suggestions on the privacytools page recently and setting privacy.resistFingerprinting back to False did resolve this issue.

That's sad though, I assume the webpage is reading my zoom level? I would assume that was something UA specific but I guess they can determine it somehow.

**Reply by kennethtran93** [reply]
You're welcome.  Glad I could be of help.
